e2e/packages/sync-test/package.json

```diff
@@ -4,7 +4,8 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "test": "vitest --run",
+    "create:db": "psql postgres://127.0.0.1/postgres -c \"CREATE DATABASE postgres_e2e;\"",
```

Comment:
is it feasible to do this in the test setup script?

Reply:
I had that initially, but it felt weird to attempt to create a database that might already exist, permissions to create a new database might be different for each user, and it only has to be executed once per machine (since the db is not dropped, just cleaned, at the end), so it felt cleaner to colocate the db creation with the place where we define the db url to use

Reply:
was just worried about the availability of `psql`